packages/math-inline/package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "@material-ui/core": "^3.9.2",
     "@pie-framework/mathquill": "^1.1.0",
+    "@pie-framework/math-validation": "^1.0.0",
```

Comment:
Shouldn't this be in controller? ("@pie-lib/math-evaluator": "2.1.6" )

Reply:
it's in controller now. works fine